content/projects.json

```diff
@@ -179,6 +179,16 @@
       "twitter": "https://twitter.com/maheshstwt",
       "live_link": "https://youtu.be/0oRW_ZetXoc",
       "stack": ["All", "React"]
+    },
+    {
+      "name": "Letam",
+      "title": "Letam",
+      "screenshot": "https://i.ibb.co/Lz0P8bt/Screenshot-127.png",
```

Comment:
Please can you explain this ?

Reply:
Sure, Right now the height of this image is long, so it must be less.

Current Image:
![image](https://user-images.githubusercontent.com/99729607/210540351-0913fddc-e052-4090-9223-ca666e68744d.png)

Expected Image:
![image](https://user-images.githubusercontent.com/99729607/210540554-883286d4-71c6-4bcc-a75f-404949d23e11.png)

### Why should we do that?
Because of how the each project carts is being displayed.
![image](https://user-images.githubusercontent.com/99729607/210540775-164c7785-f28b-49c8-8739-0c0bb19665c3.png)

Hope that makes sense